### PR TITLE
Add Locityper

### PR DESCRIPTION
**Pull request by tprodanov**
Add recipe for Locityper `v0.13.4`.
[Locityper](https://github.com/tprodanov/locityper) is a targeted genotyper for complex polymorphic genes/loci based on short and long-read whole genome sequencing.

### CODE REVIEW

**Comment by tprodanov** [comment]
@bioconda-bot please add label

File: recipes/locityper/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+if [ "$(uname)" == "Darwin" ]; then
+    export HOME="/Users/distiller"
+    export HOME=`pwd`
+fi
+
+export C_INCLUDE_PATH="$BUILD_PREFIX/include:$C_INCLUDE_PATH"
+export LIBRARY_PATH="$BUILD_PREFIX/lib:$LIBRARY_PATH"
+# export CFLAGS="-L$BUILD_PREFIX/lib"
+# export RUSTFLAGS="-L$BUILD_PREFIX/lib -lz"
```
Comment (bgruening) [inline recipes/locityper/build.sh:11]:
```suggestion
```

File: recipes/locityper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - clangdev
+    - llvm-openmp
+  host:
+    - clangdev
```
Comment (bgruening) [inline recipes/locityper/meta.yaml:25]: I don't think they are needed in the host section and build section.

**Review by bgruening — COMMENTED** [review]
Thanks! LGTM, just two small comments.

**Comment by tprodanov** [comment]
@bgruening Thank you for the recommendations, I updated the files

**Pull request merged**